types/download/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "got": "^10.7.0"
```

Comment:
This is incorrect, because `download` uses `got 8.x`: https://github.com/kevva/download/blob/a16ba04b30dafbe7d9246db93f1534320d8e0dd3/package.json#L36

Reply:
Fixed!